src/test/java/pages/HomePage.java

```diff
@@ -0,0 +1,79 @@
+package pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+
+public class HomePage extends BasePage {
+    public HomePage(WebDriver givenDriver) {
+        super(givenDriver);
+    }
+    //Web Elements;
+
+    By userAvatarIcon = By.cssSelector("img.avatar");
+    By allSongsList = By.cssSelector("li a.songs");
+
+    String newPlaylistName = "Sample Playlist";
+
+    @FindBy(xpath = "//a[contains(text(),'Oreo')]")
+    WebElement playListName;
+    @FindBy(xpath = "//button[@title= 'Delete this playlist']")
+    WebElement deletePlaylistBtn;
+
+    @FindBy(css = "div.success.show")
+    WebElement notification;
```

Comment:
You should use encapsulation of non-static and non-final fields.

If a variable is used only by other methods of that class, it should be declared as private.

I recommend:  make all the locators and WebElements private, because private keyword in java allows most restrictive access to variables and methods and offer strongest form of Encapsulation. 

For example

    ```
    private By shuffleBtnLocator = By.cssSelector(".btn-shuffle-all");
    private By firstSongLocator = By.cssSelector(".play");

```
   @FindBy(css = "img.avatar")
    private WebElement avatar
```


For example

![POM_vs_Factory_1](https://github.com/Test-Pro-Bootcamp/A55-QA-Automation-Framework/assets/104761482/9df8debd-602f-4b91-885d-59bff68a1c1e)

![POM_vs_Factory_2](https://github.com/Test-Pro-Bootcamp/A55-QA-Automation-Framework/assets/104761482/e2d89403-99e0-428e-96ae-b88825ccd3bc)